sources/in/states.hjson

```diff
@@ -51,27 +51,34 @@
             ?label_hi
             WHERE {
               SERVICE <https://query.wikidata.org/sparql> {
-              # Using nested query to ensure there is only one ?id2 value
-              {SELECT
-              ?id
-              (SAMPLE(?id2) as ?id2)
-              (SAMPLE(?iso_3166_2) as ?iso_3166_2)
-              (SAMPLE(?label_en) as ?label_en)
-              (SAMPLE(?label_hi) as ?label_hi)
-              WHERE {
-                # List of regions, whose sub-regions we want.
-                VALUES ?entity { wd:Q668 }
+                # Using nested query to ensure there is only one ?id2 value
+                {SELECT
+                ?id
+                (SAMPLE(?id2) as ?id2)
+                (SAMPLE(?iso_3166_2) as ?iso_3166_2)
+                (SAMPLE(?label_en) as ?label_en)
+                (SAMPLE(?label_hi) as ?label_hi)
+              WHERE {{
+                  # List of regions, whose sub-regions we want.
+                  VALUES ?entity { wd:Q668 }
 
-                # P150 = "contains administrative territorial entity"
-                ?entity wdt:P150 ?id .
-                ?id wdt:P300 ?iso_3166_2
+                  # P150 = "contains administrative territorial entity"
+                  ?entity wdt:P150 ?id .
+                  ?id wdt:P300 ?iso_3166_2
 
-                    OPTIONAL { ?id rdfs:label ?label_en . FILTER(LANG(?label_en) = "en") }
+                  OPTIONAL { ?id rdfs:label ?label_en . FILTER(LANG(?label_en) = "en") }
                     OPTIONAL { ?id rdfs:label ?label_hi . FILTER(LANG(?label_hi) = "hi")}
+                # Include disputed territories. See README.
+                } UNION {
```

Comment:
technically you don't need union to be part of the remote (service) query, but instead you could have done it at the outer-most level -- because you don't need wikidata service to actually query anything for you, but it is such a minor thing i wouldn't worry about it :)